demisto_sdk/commands/lint/linter.py

```diff
@@ -1306,3 +1346,285 @@ def _get_commands_list(self, script_obj: dict):
+    ) -> bool:
+        """
+        Gets a native image name (flag) and checks if it supports the integration/script that lint runs on.
+
+        Args:
+            native_image (str): Name of the Native image to run on.
+            supported_native_images (set): A set including the names of the native images that support the
+                                           script/integration that lint runs on.
+            script_id (str): The ID of the integration/script that lint runs on.
+        Returns (bool): True - if the native image supports the integration/script that lint runs on.
+                        False - Otherwise.
+        """
+        if native_image not in supported_native_images:
+            # Integration/Script isn't supported by the requested native image
+            logger.info(
+                f"Skipping checks on docker for {native_image} - {script_id} is not supported by the "
+                f"requested native image: {native_image}."
+            )
+            return False
+
+        return True
+
+    def _get_native_image_name_from_config_file(
+        self,
+        docker_image_flag: str,
+    ) -> Union[str, None]:
+        """
+        Gets a native docker image flag and returns its mapped native image name from the
+        'docker_native_image_config.json' file.
+
+        If the requested docker image flag doesn't have a mapped native image in the config file - write to the logs.
+
+        Args:
+            docker_image_flag (str): Requested docker image flag.
+        Returns (None): None
+        """
+        native_image_config = (
+            NativeImageConfig()
+        )  # parsed docker_native_image_config.json file (a singleton obj)
+
+        native_image = native_image_config.flags_versions_mapping.get(docker_image_flag)
+        if native_image:
+            return native_image
+
+        else:
+            # The requested native image doesn't exist in the native config file
+            logger.info(
+                f"Skipping checks on docker for '{docker_image_flag}' - The requested native image: "
+                f"'{docker_image_flag}' is not supported. For supported native image versions please see:"
+                f" 'Tests/docker_native_image_config.json'."
+            )
+            return None
+
+    def _get_dev_native_image(self, script_id: str) -> Union[str, None]:
+        """
+        Gets the development (dev) native image, which is the latest tag of the native image from Docker Hub.
+        Args:
+            script_id (str): The ID of the integration/script that lint runs on.
+        Returns: The reference of the dev native image.
+        """
+        log_prompt = f"{self._pack_name} - Get Dev Native Image"
+        logger.info(f"{log_prompt} - Started")
+
+        # Get the latest tag of the native image from Docker Hub
+        latest_tag = DockerImageValidator.get_docker_image_latest_tag_request(
+            NATIVE_IMAGE_DOCKER_NAME
+        )
```

Comment:
can you please log what the latest tag that we got?

Reply:
I have a log with the latest docker image in line 1607. 
Should I add another one here?